website/client/src/components/challenges/index.vue

```diff
@@ -1,33 +1,37 @@
 <template>
   <div class="row">
-    <secondary-menu class="col-12">
-      <router-link
-        class="nav-link"
-        :to="{name: 'myChallenges'}"
-        :class="{'active': $route.name === 'myChallenges'}"
-      >
-        {{ $t('myChallenges') }}
-      </router-link>
-      <router-link
-        class="nav-link"
-        :to="{name: 'findChallenges'}"
-        :class="{'active': $route.name === 'findChallenges'}"
-      >
-        {{ $t('findChallenges') }}
-      </router-link>
-    </secondary-menu>
+    <navbar
+      class="secondary-menu col-12"
+      :items="tabs"
+      router-links
+    />
     <div class="col-12">
       <router-view />
     </div>
   </div>
 </template>
 
 <script>
-import SecondaryMenu from '@/components/secondaryMenu';
+import Navbar from '@/components/ui/simpleNavbar';
 
 export default {
   components: {
-    SecondaryMenu,
+    Navbar,
+  },
+  computed: {
```

Comment:
I think this can be a normal data property instead of a computed one because the data is static